packages/language-service/ivy/completions.ts

```diff
@@ -209,7 +209,19 @@ export class CompletionBuilder<N extends TmplAstNode|AST> {
         return undefined;
       }
 
-      const replacementSpan = makeReplacementSpanFromAst(this.node);
+      let includeDot = false;
+      if (!(this.node.receiver instanceof ImplicitReceiver) &&
+          options?.includeCompletionsWithInsertText &&
+          options.includeAutomaticOptionalChainCompletions) {
```

Comment:
```suggestion
          options.includeAutomaticOptionalChainCompletions !== false) {
```
Based on the documentation for this option, it has to explicitly be set to `false` to disable it.



> Unless this option is `false`, or `includeCompletionsWithInsertText` is not enabled,
> member completion lists triggered with `.` will include entries on potentially-null and potentially-undefined
> values, with insertion text to replace preceding `.` tokens with `?.`.
>